Common/GPU/OpenGL/thin3d_gl.cpp

```diff
@@ -582,6 +582,15 @@ OpenGLContext::OpenGLContext() {
 		}
 	}
 
+	bugs_.Infest(Bugs::RASPBERRY_SHADER_COMP_HANG);
```

Comment:
Is this intended or left from debugging?

Reply:
Left from debugging, doh